lib/Raven/Middleware/UserInterfaceMiddleware.php

```diff
@@ -40,8 +40,6 @@ public function __invoke(Event $event, callable $next, ServerRequestInterface $r
             $userContext['ip_address'] = $request->getHeaderLine('REMOTE_ADDR');
         }
 
-        $event = $event->withUserContext($userContext);
```

Comment:
Is this lost?

Reply:
No it's not, I'm using the `offsetSet` magic method of the `Context` class a few lines above